include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -1002,13 +990,10 @@ __pattern_unique_copy(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&& __exec
 }
 
 template <typename _Name>
-class copy_back_wrapper
-{
-};
+struct copy_back_wrapper;
+
 template <typename _Name>
-class copy_back_wrapper2
-{
-};
+struct copy_back_wrapper2;
```

Comment:
id recommend changing this to `copy_back2_wrapper` for consistency, or encoding more meaningful information in the name instead of `2`.